### PR TITLE
Update Spring Boot to 1.5.15

### PR DESCRIPTION
**Pull request by geoand**

### CODE REVIEW

**Comment by geoand** [comment]
@metacosm this is the new PR for `1.5.15`

**Comment by gastaldi** [comment]
[test]

**Comment by geoand** [comment]
@metacosm The cache booster was updated, could you take a look please?

**Comment by metacosm** [comment]
[test]

**Comment by gastaldi** [comment]
[test]

File: metadata.yaml [hunk 1]
```diff
@@ -80,247 +80,333 @@ missions:
   metadata:
     level: advanced
 runtimes:
-- id: vert.x
```
Comment (gastaldi) [inline metadata.yaml]: Can you revert the changes in this file?

**Comment by gastaldi** [comment]
I haven't compared the contents yet, but if the format is the only thing that changed that may be fine (and hopefully it won't be messed up the next time your automatic process kicks in I suppose)
